src/main/java/com/esri/samples/map/create_and_save_map/AuthenticationDialog.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.map.create_and_save_map;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.TextField;
+
+import com.esri.arcgisruntime.security.OAuthConfiguration;
+
+/**
+ * Custom dialog for getting an OAuthConfiguration.
+ */
+class AuthenticationDialog extends Dialog<OAuthConfiguration> {
+
+  @FXML
+  private TextField portalURL;
```

Comment:
Which way do we want these to look, some samples I been doing have it all on one line.
`@FXML private TextField portalURL;`